libtest.sh

```diff
@@ -366,9 +366,9 @@ runCoBG(){
         ./conode -d $DBG_SRV -c co$nb/private.toml server >& "$COLOG$nb.log"
       fi
       touch "$COLOG$nb.log.dead"
-    ) &
+    ) 2>/dev/null &
     # This makes `pkill conode` not outputting errors here
-    disown
+    disown %1
```

Comment:
Seems I missed how to properly disown the process here - what is the %1 doing?

Reply:
I thought it was required by the disown docs, but not I see `If no jobspec is present, and neither the -a nor the -r option is supplied,  the  current job is used` so perhaps it was always working correctly.

The problem I was trying to solve was that bash was complaining about the child process dying, but that was solved by throwing the stderr from the subshell away (the stderr from the conode is already handled above). So I will remove the %1.